index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+
+  <body>
+    
+    <div id="header">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="">      
+     <div id="hehe"> 
+      <nav id="nav-bar">
+      <ul>
+        <li>
+          <a href="#features" class="nav-link"> Features</a>
+        </li>
+        <li>
+          <a href="#how-it-works" class="nav-link"> How It Works</a>
+        </li>
+        <li>
+          <a href="#pricing" class="nav-link"> Pricing</a>
+        </li>
+      </ul>
+      </nav>
+      </div>
+    </div>
+    <div id="above">
+      <h2>Handcrafted, home-made masterpieces</h2>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input type="email" name="email"  id="email" placeholder="Enter your email address" required>
+        <br>
+        <input type="submit" id="submit" value="Get Started">
+
+      </form>
+    </div>
+    <section id="features">
+      <div class="grid">
+        <div class="icon">
+          <i class="fa fa-fire"></i>
+        </div>
+        <div class="more-info">
+          <h2>Premium Materials</h2>
+          <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. </p>
+        </div>
+      </div>
+
+      <div class="grid">
+        <div class="icon">
+          <i class="fa fa-truck"></i>
+        </div>
+        <div class="more-info">
+          <h2>Fast Shipping</h2>
+          <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. </p>
+         
+        </div>
+      </div>
+      <div class="grid">
+        <div class="icon">
+          <i class="fa fa-battery-full"></i>
+        </div>
+        <div class="more-info">
+          <div id="special">
+          <h2>Quality Assurance</h2>
+          <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.  </p>
+         </div>
+        </div>
+      </div>
+
+      
+     
+    </section>
+
+    
+      <video id= "video" controls>
+        <source src="https://youtu.be/y8Yv4pnO7qc" type="video/mp4">
+      </video>
```

Comment:
**kindly remove all unnecessary empty lines, so that your code looks professional.**
### Please,  put the video in a div. 
**Its symbolic purpose is to divide a section/portion of the page so that a separate style can be added or applied to it.**